src/main/java/net/ltxprogrammer/changed/VariantCheck/PatienceCompatibility.java

```diff
@@ -0,0 +1,70 @@
+package net.ltxprogrammer.changed.VariantCheck;
+
+import io.github.edwinmindcraft.origins.api.OriginsAPI;
+import io.github.edwinmindcraft.origins.api.capabilities.IOriginContainer;
+import io.github.edwinmindcraft.origins.api.origin.Origin;
+import io.github.edwinmindcraft.origins.api.origin.OriginLayer;
+import net.ltxprogrammer.changed.Changed;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.server.level.ServerPlayer;
+
+import java.util.Objects;
+
+enum VariantType {
+    LATEX_RESISTANCE, COMPATIBILITY, DEFAULT
+}
+
+public class PatienceCompatibility {
+    private final ServerPlayer player;
+    private boolean conditionValue = false;
+    private VariantType variantType = VariantType.DEFAULT;
+
+    public PatienceCompatibility(ServerPlayer player) {
```

Comment:
Using a whole class like this to check origin compatibility is really inconvenient, because it has to be initialized every time you need to check it. `ChangedCompatibility.isPlayerUsedByOtherMod()` is a one liner that checks everything in one function.